dist/templates/banner-cookie-declaration-consent/banner-cookie-declaration-consent.html

```diff
@@ -72,7 +73,7 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
   <div class="coi-consent-banner__base" id="coiConsentBannerBase">
     <div class="coi-consent-banner__consent-area">
-      <button onclick="CookieInformation.submitConsent()" class="coi-consent-banner__agree-button">
+      <button onclick="CookieConsent.dialog.submitConsent();" class="coi-consent-banner__agree-button">
```

Comment:
We should use new function from library API for this logic (submitAllCategories) instead of this legacy.